concordia/admin/__init__.py

```diff
@@ -310,13 +310,13 @@ class ItemAdmin(admin.ModelAdmin):
 
     actions = (publish_item_action, unpublish_item_action)
 
-    def get_deleted_objects(self, objs, request):
+    def get_deleted_objects(self, objs, request):  
         deleted_objects = [str(obj) for obj in objs]
-        model_count = {'Items': len(deleted_objects)}
-        perms_needed = []
+        perms_needed = set()
         protected = []
+        model_count = {'Items': len(deleted_objects)}
```

Comment:
This could be efficiently calculated as something like this — note the use of `verbose_name_plural` as per [the ModelAdmin docs](https://docs.djangoproject.com/en/3.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.get_deleted_objects) — to calculate the totals using one query per model rather than n-per-instance:

```python
model_count = {
    Item._meta.verbose_name: len(objs),
    Asset._meta.verbose_name_plural: Asset.objects.filter(item__in=objs).count(),
    Transcription._meta.verbose_name_plural: Transcription.objects.filter(asset__item__in=objs).count(),
}
```

Reply:
done